eclide/ChildBuilderFrame.cpp

```diff
@@ -404,7 +408,27 @@ class CBuilderFrame :
             IAttributeVector attrs;
             Dali::CEclExceptionVector errors;
             IAttributeBookkeep attrProcessed;
-            attr->PreProcess(PREPROCESS_SUBMIT, _ecl, attrs, attrProcessed, errors);
+            PREPROCESS_TYPE processType = PREPROCESS_UNKNOWN;
+            switch (action) {
+            case Dali::WUActionGenerate:
+                if (!isSaved)
+                    m_dlgview.DoGenerate();
+                processType = PREPROCESS_GENERATE;
+                break;
+            case Dali::WUActionCustom1:
+                processType = PREPROCESS_CUSTOM1;
+                break;
+            case Dali::WUActionCustom2:
+                processType = PREPROCESS_CUSTOM2;
+                break;
+            case Dali::WUActionCustom3:
+                processType = PREPROCESS_CUSTOM3;
+                break;
+            case Dali::WUActionCustom4:
+                processType = PREPROCESS_CUSTOM4;
+                break;
```

Comment:
@rpastrana no, it does not. Line 411 sets the default value when the variable is declared: `PREPROCESS_TYPE processType = PREPROCESS_UNKNOWN;`